omod/src/main/webapp/resources/partials/daterangepicker.html

```diff
@@ -1,13 +1,14 @@
-<div class="date-range-picker-div" class="inlineBox">
-    <p> {{headermessage}} </p>
-    <span class="angular-datepicker">
-        <input class="appoinmentrangepicker" type="text" is-open="startDateOptions.opened" ng-model="startDate" min="{{startDateMin}}" max="endDate" show-weeks="false" datepicker-popup="dd MMM yyyy" readonly/>
-        <i class="icon-calendar small add-on" ng-click="startDateOptions.open($event)" ></i>
-        <a class="date-range-picker-clear-link add-on" ng-click="startDateOptions.clear($event)">{{ clearLinkText }}</a>
+<div class = "col-xs-5">
```

Comment:
what was the purpose of that bootstrap class ?? it looks like doesnt do any thing

Reply:
Let me remove that class, anything else minus that